### DISCUSSION TITLE
Welcome to dAcademy Discussions!

### DISCUSSION THREAD

**Discussion by Innkeeping** [MetaFam/dAcademy#45] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by MxianD** [comment]
Hi! I am very interested in your project. Actually, I have some experience in this. Do you know how to join you guys? I can be a developer or even a co-founder.

**Reply by dysbulic** [reply]
I apologize. I'm *just* seeing this message.

If you're interested, we do semi-regular pair programming sessions & you're welcome to join in.

 I'm @dysbulic on Discord. My DMs are open. Just drop me a line & I'll work out the scheduling for a programming session.
